workflow/scripts/error_rate.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+import pandas as pd
+import yaml
+
+def min_err_rate():
+    try:
```

Comment:
Again maybe move this down. I think that would make me as a reader more easily see what makes the function work.